src/style.css

```diff
@@ -0,0 +1,113 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+/* @font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+} */
+
+/* html {
+  font-family: "Avenir", Arial, sans-serif;
+} */
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-weight: 300px;
+}
+
+.form-big {
+  position: relative;
+  min-width: 200vh;
+  margin-top: 20px;
+  margin-right: 8px;
+  margin-left: 8px;
+  background-position: center;
+}
+
+.img-big {
+  position: absolute;
+  padding-left: 26px;
+  top: 25px;
+  bottom: 26px;
+  height: 19px;
+  width: 19px;
+}
+
+.input-big::placeholder {
+  color: #3d4e61;
+}
+
+.input-big {
+  width: 100%;
+  height: 70px;
+  display: flex;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
```

Comment:
```suggestion
  font-family: inherit;
```

Reply:
for some reason it does not work for me so, so I use the second way, if as you suggest the style does not change, respectively, the style is not according to the layout
right here, as you recommend.
![image](https://user-images.githubusercontent.com/119611480/209461846-60b2f64b-a84d-4f25-b3b3-dccf91a10779.png)
right here, as I do
![image](https://user-images.githubusercontent.com/119611480/209461911-d143dd27-107e-4fa2-a743-7e1f33d853d3.png)

Reply:
If you want that `inherit` rule will work, you have to add `font-family` rule to the `html` or `body` selector.

Reply:
fix

Reply:
corrected